articles/references/domain-designer/associations/associations.md

```diff
@@ -3,36 +3,34 @@ uid: references.domain-designer.associations
 ---
 # Domain Designer Associations
 
-Intent Architect's Domain Designer was originally built with the idea of being able to model Class Diagrams similar to how the UML specification would have it.
-Once a `Class` has been defined by its `members` and `operations` it would be useful to also model how it would relate with another class using some visual arrow and some symbol notation denoting the kind of relationship between the two `Classes`.
-Wikipedia provides some information regarding the concept of [Relationships](https://en.wikipedia.org/wiki/Class_diagram#Relationships) if you want to find out more.
```

Comment:
By specifying "originally built", it kind of makes me ask the question of "how is it now then?". So I've changed this to merely state Intent Architect's capabilities.